packet.go

```diff
@@ -584,6 +584,13 @@ func (p *sshFxpReadPacket) UnmarshalBinary(b []byte) error {
 	return nil
 }
 
+func (p *sshFxpReadPacket) getDataSlice() []byte {
+	dataLen := clamp(p.Len, maxTxPacket)
+	// we allocate a slice with a bigger capacity so we avoid a new allocation in MarshalBinary and in sendPacket
+	// we need 9 bytes in MarshalBinary and 4 bytes in sendPacket
+	return make([]byte, dataLen, dataLen+13)
```

Comment:
Writing this as `+9+4` makes the magic number parallel the code comment more directly, and we don’t have to do arithmetic in our head every time to ensure the numbers line up.